hibernate-reactive-core/src/main/java/org/hibernate/reactive/session/impl/ReactiveSessionImpl.java

```diff
@@ -1721,7 +1721,7 @@ public ReactiveConnection getReactiveConnection() {
 
 	@Override
 	public void close() throws HibernateException {
-		throw new UnsupportedOperationException( "Non reactive close method called. Use 'reactiveClose()' instead." );
+		throw new UnsupportedOperationException( "Non-reactive 'close()' method called. Use 'reactiveClose()' instead." );
```

Comment:
Actually, we now have a specific error in the Log now: https://github.com/hibernate/hibernate-reactive/blob/main/hibernate-reactive-core/src/main/java/org/hibernate/reactive/logging/impl/Log.java#L243

Reply:
OK, but we can fix that up in a subsequent commit.